synfig-studio/src/gui/app.h

```diff
@@ -32,7 +32,7 @@
 #include <ETL/smart_ptr>
 
 #include <gtkmm/box.h>
-#include <gtkmm/main.h>
+// #include <gtkmm/main.h>
```

Comment:
You can safely delete this line.

Reply:
Understood ! will get removed in next commit :)